go.mod

```diff
@@ -6,10 +6,11 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/pkg/sftp v1.13.4
 	github.com/sirupsen/logrus v1.8.1
-	github.com/urfave/cli/v2 v2.6.0
+	github.com/urfave/cli/v2 v2.8.0
 )
 
 require (
+	github.com/antzucaro/matchr v0.0.0-20210222213004-b04723ef80f0 // indirect
```

Comment:
This seems GPL, not compatible with Apache License 2.0: https://github.com/antzucaro/matchr/blob/master/COPYING.txt

Reply:
Opened an issue https://github.com/urfave/cli/issues/1404